apps/backend/src/sales/sales.service.ts

```diff
@@ -17,13 +18,13 @@ export class SalesService {
     try {
       await (
         await this.firehoseService.putRecord(
-          event as unknown as Record<string, string>
+          parseGarms(event) as unknown as Record<string, string>
         )
       ).promise();
```

Comment:
The event should go to the DB here, it can be the next PR though, we can send it to the void for now.

Reply:
Haha, no ideas why I decided call parseGarms here. I wanted to return the parsed garms as some sort of response, but I am doing that already outside of the try/catch. 

I intentionally did not include the db update here as we are currently ingesting the sales files over sftp, and if we started manually adding in test data here it might mess things up ?